inc/rest.php

```diff
@@ -401,6 +401,18 @@ public function register_service( WP_REST_Request $request ) {
 			);
 
 		}
+
+		if ( $user === 'site_exists' ) {
+			return new WP_REST_Response(
+				[
+					'data'    => null,
+					'message' => sprintf( __( 'Error: This site has been previously registered. You can try again %1$shere%2$s ', 'optimole-wp' ), '<a href="https://dashboard.optimole.com/register" target="_blank"> ', '</a>' ),
```

Comment:
@HardeepAsrani, what's the advantage of redirecting to the register page?

Reply:
Followed the same notice pattern for the notice if the user is already registered. In case you want to try to log in as there is no other way to log in using a username/password from the plugin.

Reply:
@HardeepAsrani my point is more if we don't allow people to register with the same domain, what's the point of redirecting them to the register page?

Reply:
@selul Got it. I'll replace it with the login link and modify the message accordingly.